tests/phpunit/api/v3/PaymentTest.php

```diff
@@ -406,4 +406,61 @@ public function testUpdatePayment() {
     ));
   }
 
+  /**
+   * Test create payment api for paylater contribution
+   */
+  public function testCreatePaymentPayLater() {
+    $contributionParams = array(
+      'total_amount' => 100,
+      'currency' => 'USD',
+      'contact_id' => $this->_individualId,
+      'financial_type_id' => 1,
+      'contribution_status_id' => 2,
+      'is_pay_later' => 1,
+    );
+    $contribution = $this->callAPISuccess('Contribution', 'create', $contributionParams);
+
+    //Create partial payment
+    $params = array(
+      'contribution_id' => $contribution['id'],
+      'total_amount' => 100,
+    );
+    $payment = $this->callAPIAndDocument('Payment', 'create', $params, __FUNCTION__, __FILE__);
```

Comment:
I think I mentioned this on the other PR - you don't want to keep documenting - because only one example will be retained for each entity-action unless you add other params - so creating & example & then overwriting it is just noise.